ui/packages/consul-ui/app/templates/dc/services/show/topology.hbs

```diff
@@ -27,66 +27,107 @@ as |route|>
+            (array 'no-dependencies' (or false (and topology.noDependencies (can 'use acls'))))
+            (array 'acls-disabled' (or false (and topology.noDependencies (not (can 'use acls')))))
+          ))
+        as |notices|}}
+
+          {{! Make an array only of truthy values to we know how many notices are enabled }}
+          {{#let (without false
+            (values notices)
+          ) as |noticesEnabled|}}
+
+            {{! Render any enabled notices }}
+            {{#each-in notices as |prop enabled|}}
+              {{#if enabled}}
+                <disclosure.Details
+                  @auto={{false}}
+                as |details|>
+                  <Notice
+                    class="topology-metrics-notice"
+                    id={{details.id}}
+                    data-test-notice={{prop}}
+                    @type={{if (contains prop (array 'filtered-by-acls' 'no-dependencies'))
+                      'info'
+                      'warning'
+                    }}
+                  as |notice|>
+                    <notice.Header>
+                      <h3>
+                        {{compute (fn route.t 'notice.${prop}.header'
+                          (hash
+                            prop=prop
+                          )
+                        )}}
+                      </h3>
+                    </notice.Header>
+                    {{#if disclosure.expanded}}
+                      <notice.Body>
+                        <p>
+                          {{compute (fn route.t 'notice.${prop}.body'
+                            (hash
+                              prop=prop
+                            )
+                          )}}
+                        </p>
+                      </notice.Body>
+                    {{/if}}
+                    {{#let
+                      (compute (fn route.t 'notice.${prop}.footer'
+                        (hash
+                          route_intentions=(href-to 'dc.services.show.intentions')
+                          prop=prop
+                          htmlSafe=true
+                        )
+                      ))
+                    as |footer|}}
+                      {{#if (and disclosure.expanded (not-eq prop 'filtered-by-acls'))}}
+                        <notice.Footer>
+                          {{footer}}
+                        </notice.Footer>
+                      {{/if}}
+                    {{/let}}
+                  </Notice>
+                </disclosure.Details>
+              {{/if}}
+            {{/each-in}}
+
+            {{! If more than 2 notices are enabled let the user close them }}
+            {{#if (gt noticesEnabled.length 2)}}
+              <disclosure.Action
+                {{on 'click' disclosure.toggle}}
+              >
+              {{compute (fn route.t 'notices.${expanded}'
+                (hash
+                  expanded=(if disclosure.expanded 'close' 'open')
+                )
+              )}}
```

Comment:
Heres an example of the `route.t` thing, its a little involved here as I'm using template literal style replacement to avoid using `concat` all the time. But usually it would be just :

```
{{compute (fn route.t 'notices.header')}}
```

I think in later versions of Ember I'll be able to get rid of the `compute` and the `fn`. Anyway thoughts as always appreciated but do keep the "this is to do relative keying" usecase in mind, ty!